### ISSUE TITLE
Design tweaks

### ISSUE THREAD

**Issue by alyblenkin** [getodk/web-forms#172]
Sadiq and I discussed making small iterative improvements every month. So we will keep a running list and then evaluate, what we want to prioritize when the time comes. 

**Errors**
- [ ] Error banner: "Fix errors" button needs to move to the right slightly (so the top, bottom, and right all have equal padding) 

### Will separate out into individual stories
-  [Add chips for multiselects](https://primevue.org/multiselect/#chips)
- Shorter textfields and for multiple-line appearances (is it possible to use fields appropriate to the length of the input, like names, numeric questions etc).

**Comment by alyblenkin** [comment]
We are going to break this into new stories

**Comment by latin-panda** [comment]
@alyblenkin I wonder if we can mark some of these are completed?

**Comment by alyblenkin** [comment]
Yes, thanks for cleaning up these old issues! We've clearly wanted to do this for some time! Do we have an issue for adding the action to the toast?

**Comment by latin-panda** [comment]
> Do we have an issue for adding the action to the toast?

@alyblenkin maybe this one: https://github.com/getodk/web-forms/issues/331
